### ISSUE TITLE
EVSS PCIU Migration: Test Plan Step 3 Form VA21p530 - initialize_contact_information

### ISSUE THREAD

**Issue by RachalCassity** [department-of-veterans-affairs/va.gov-team#76170]
### Test Plan: Step 3 Replace PCIU

`FormData.initialize_contact_information `prefills contact information with PCIU data.  
Coverband is showing a lot of data. This will have to be tested when migrating to VAProfile. PCIU references need to be removed. 

```
  def initialize_contact_information
    opt = {}
    opt.merge!(vets360_contact_info_hash) if vet360_contact_info
    opt[:address] ||= user_address_hash
    opt[:email] ||= extract_pciu_data(:pciu_email)
    if opt[:home_phone].nil?
      opt[:home_phone] = pciu_primary_phone
      opt[:us_phone] = pciu_us_phone
    end
    format_for_schema_compatibility(opt)
    FormContactInformation.new(opt)
  end
```

## Tasks

- [x] Determine if the /claimant-information endpoint still active? 
- [ ] Ensure initialize_contact_information is returning  vet360_contact_info
- [ ] Remove initialize_contact_information PCIU calls

**Comment by RachalCassity** [comment]
Task  **Determine if the /claimant-information endpoint still active?**
The claimant-information url is still relevant. A form, that uses this endpoint, was created six days ago in this [PR](https://github.com/department-of-veterans-affairs/vets-api/pull/16215/files).

**Comment by LindseySaari** [comment]
@RachalCassity I was able to get ahold of the OpenAPI docs for the Contact Info. Just load it into https://editor.swagger.io/ Otherwise we have to be on the network to access the docs

**Comment by LindseySaari** [comment]
@RachalCassity I was able to get ahold of the OpenAPI docs for the Contact Info. Just load it into https://editor.swagger.io/ Otherwise we have to be on the network to access the docs

**Comment by jennb33** [comment]
Per @RachalCassity : Setup a remote branch in the vets-api. Waiting for the meeting with VA Profile to answer questions.

**Comment by RachalCassity** [comment]
Reached out to Veteran Facing Forms in this [slack thread](https://dsva.slack.com/archives/C044AGZFG2W/p1713542803179949)

**Comment by RachalCassity** [comment]
Waiting to hear back from VFF. Form Profile Spec is illegible and the prefill settings do not accurately follow what the settings are in other environments. I'm going through each form and fixing the prefill data and removing unnecessary prefill methods.

**Comment by RachalCassity** [comment]
Logs were added to ensure VAProfile Contact Info has been merged for all users. Logs will be monitored for the next sprint before removing PCIU references.

**Comment by RachalCassity** [comment]
These [logs](https://vagov.ddog-gov.com/logs?query=%22User%20Vet360%20Contact%20Info%2C%20Address%3F%20true%2C%20Email%3F%20false%2C%20Phone%3F%20false%22%20env%3Aeks-prod%20&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&from_ts=1713318517000&to_ts=1713923317000&live=true) are showing that user data has not been merged into VAProfile. 
PCIU data will have to be merged to VAProfile before we can depreciate PCIU.

**Comment by RachalCassity** [comment]
@jennb33 @LindseySaari 
Ticket created for VAProfile: https://github.com/department-of-veterans-affairs/va.gov-team/issues/81208

**Comment by jennb33** [comment]
8/5 update: This will remain a Platform ticket, but VFS teams need to do the testing and custom implementation and or/migration of these methods.

**Comment by AshleyGuerrant** [comment]
Not completed during Sprint 14 because the dev. was on support which took priority. Moving to Sprint 15.
